core/src/com/unciv/ui/EmpireOverviewScreen.kt

```diff
@@ -177,6 +177,10 @@ class EmpireOverviewScreen : CameraStageBaseScreen(){
             greatPeopleTable.add(greatPersonPoints[entry.key]!!.toInt().toString()+"/"+pointsToGreatPerson)
             greatPeopleTable.add(greatPersonPointsPerTurn[entry.key]!!.toInt().toString()).row()
         }
+        val pointsForGreatGeneral = playerCivInfo.greatPeople.greatGeneralPoints.toInt().toString()
+        val pointsForNextGreatGeneral = playerCivInfo.greatPeople.pointsForNextGreatGeneral.toInt().toString()
+        greatPeopleTable.add("Great General".tr())
```

Comment:
Is "Great General" in the translation file? If not, it should be added

Reply:
Yes. It is already in translation file.